build_tools/patch_monorepo.py

```diff
@@ -141,6 +141,7 @@ def main(argv):
             "rocSPARSE",
             "rocThrust",
             "MIOpen",
+            "composable_kernel",
```

Comment:
I'm not sure how this script is used, but if it affects Windows we may want to skip this given that CK fails to checkout on Windows.